frontend/src/pages/org/collection-detail.ts

```diff
@@ -553,19 +534,47 @@ export class CollectionDetail extends BtrixElement {
           (collection) =>
             this.isEditingDescription
               ? html`
-                  <btrix-markdown-editor
-                    initialValue=${collection.description || ""}
-                    placeholder=${msg("Tell viewers about this collection")}
-                    maxlength=${4000}
-                  ></btrix-markdown-editor>
+                  <div class="flex justify-center leading-relaxed">
+                    <div class="w-full md:max-w-[783px]">
```

Comment:
Where does the `783px` value come from? Can we use one of the CSS variables here?

Reply:
Comes from being sloppy while trying to figure out a comfortable value 🙃 

Do we have a global value that we use for max-width with text?  On the site this is part of the `prose` tailwind typography setting (which I would rather set separately because it's somewhat regularly overruled)...  Maybe we should be using that here too?